pyrsistent/_pbag.py

```diff
@@ -138,6 +138,89 @@ def __lt__(self, other):
     __gt__ = __lt__
     __ge__ = __lt__
 
+    # Multiset-style operations from collections.Counter
+
+    # It would be cool to implement these by directly calling the methods
```

Comment:
Yes, I agree. I don't have any good suggestions for how to do it. The current implementation looks good though so please just remove these comments.